android/default.nix

```diff
@@ -94,6 +94,12 @@ in rec {
 
     , additionalDependencies ? ""
 
+    , nativeDependencies ? {}
+      # Allows to copy native .so libraries into APK. Example:
+      # nativeDependencies = nixpkgs: {
+      #   "libsodium.so" = "${nixpkgs.libsodium}/lib/libsodium.so";
```

Comment:
Do we need an attribute set, or might we just have a list and have the files keep their original names?

Reply:
Indeed, after some practice with renaming libraries such way, it is no point in renaming as you have already linked against the original name.